pyproject.toml

```diff
@@ -73,6 +73,7 @@ jinja2 = "<3.1.0"
 greenlet = "^3.0.0"
 jwcrypto = "^1.5.0"
 typos = "^1.16.11"
+pytest-mock = "^3.11.1"
```

Comment:
something is strange, this line, and lock file changes are already in main, so I'm not sure why they are showing as changes here. 

There seem to be other "changes" showing up here as well, like the `write`/`write_data` changes, I'm not sure what's going on exactly.

Reply:
Looks like some additional content was added / squashed-into the branch when updating from https://github.com/hvac/hvac/commit/8ecce9615ae6b9fd558ff55036668af95aefd2c2 to https://github.com/hvac/hvac/commit/fa32787b181e1b919ba6aadc09787b2cb22c5200